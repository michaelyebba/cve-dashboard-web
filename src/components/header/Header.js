import React from 'react'

let Header = (props) => {
    return (
        <header id="header">
        <div class="container">
          <nav>
            <a href="#" class="logo"><i class="fas fa-exclamation-triangle"></i>CVE Dashboard</a>
            <ul class="reports">
              <li class="active"><a href="#">Most Recent</a></li>
              <li><a href="#">HIGH</a></li>
              <li><a href="#">MEDIUM</a></li>
              <li><a href="#">LOW</a></li>
            </ul>
          </nav>
        </div>
      </header>
    )
}

export default Header;