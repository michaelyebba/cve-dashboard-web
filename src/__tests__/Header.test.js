import React from 'react';
import {HashRouter} from 'react-router-dom'
import renderer from 'react-test-renderer';
import Header from '../components/header/Header';

it('renders correctly', () => {
  const tree = renderer
    .create(
        <HashRouter>
            <Header/>
        </HashRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});