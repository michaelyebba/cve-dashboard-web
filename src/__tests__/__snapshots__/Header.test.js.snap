// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  id="header"
>
  <div
    className="container"
  >
    <nav>
      <a
        className="logo"
        href="#/"
        onClick={[Function]}
      >
        <i
          className="fas fa-exclamation-triangle"
        />
        TOP 10 CVE Dashboard
      </a>
      <ul
        className="reports"
      >
        <li>
          <a
            aria-current="page"
            className="active"
            href="#/"
            onClick={[Function]}
            style={Object {}}
          >
            ALL
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            href="#/cves/severity/HIGH"
            onClick={[Function]}
          >
            HIGH
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            href="#/cves/severity/MEDIUM"
            onClick={[Function]}
          >
            MEDIUM
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            href="#/cves/severity/LOW"
            onClick={[Function]}
          >
            LOW
          </a>
        </li>
      </ul>
    </nav>
  </div>
</header>
`;
